molgenis-data-mapper/src/main/java/org/molgenis/data/mapper/controller/MappingServiceController.java

```diff
@@ -122,6 +138,8 @@ public String addMappingProject(@RequestParam("mapping-project-name") String nam
 		MappingProject newMappingProject = mappingService.addMappingProject(name, getCurrentUser(), targetEntity);
 		// FIXME need to write complete URL else it will use /plugin as root and the molgenis header and footer wont be
 		// loaded
+
+		// FIXME redirect puts all model elements into the url
 		return "redirect:/menu/main/mappingservice/mappingproject/" + newMappingProject.getIdentifier();
```

Comment:
That'll clear away the extra parameters from the URL but a reload will resubmit the post then.